controllers/schemas/user.js

```diff
@@ -0,0 +1,100 @@
+const validator = require('../../utils/schemaValidator')
+// const ObjectId = require('mongodb').ObjectID
+
+const getOne = {
+  id: {
+   // isMongoId: true,
+    isMongoId: true,
+    notEmpty: true,
+    errorMessage: 'This is not a correct MongoID'
+  }
+}
+
+const post = {
+  name: {
+    notEmpty: true,
+    errorMessage: 'The field should contain your name'
+  },
+  office: {
+    notEmpty: true,
+    errorMessage: 'The field should containt the office name'
+  },
+  password: {
+    notEmpty: true,
+    errorMessage: 'Invalid password'
+  },
+  email: {
+    notEmpty: true,
+    isEmail: true,
+    errorMessage: 'Invalid email'
+  },
+  phone: {
+    notEmpty: true,
+    isNumber: false,
+    errorMessage: 'The phone is in the incorrect format'
+  },
+  role: {
+    notEmpty: true,
+    errorMessage: 'You need to fullfill the field with your role'
+  },
+  slackUser: {
+    notEmpty: true,
+    errorMessage: 'You should type your slack user'
+  },
+  isActive: {
+    isBoolean: {
+      errorMessage: 'Should be a boolean'
+    }
+  }
+
+}
+
+const put = {
+  name: {
+    notEmpty: true,
```

Comment:
what if i want to modify the user but i do not want to modifiy the name? just office for example? this will require that i must to send it.

Reply:
the problem was solved, also is on the new pull request, thanks